lib/net/http/response.rb

```diff
@@ -362,26 +377,30 @@ def read_body(dest = nil, &block)
     @body
   end
 
-  # Returns the full entity body.
+  # Returns the string response body;
+  # note that repeated calls for the unmodified body return a cached string:
   #
-  # Calling this method a second or subsequent time will return the
-  # string already read.
+  #   path = '/todos/1'
+  #   Net::HTTP.start(hostname) do |http|
+  #     res = http.get(path)
+  #     p res.body
+  #     p res.body.object_id
+  #     p res.body.object_id
```

Comment:
I'm not a fan of examples that use the object ID, I think we should just get rid of it.

Reply:
Removed.